addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/InformationDisclosureSuspiciousCommentsScanRuleUnitTest.java

```diff
@@ -97,14 +102,39 @@ void shouldReturnExpectedMappings() {
     }
 
     @Test
-    void shouldAlertOnSuspiciousCommentInJavaScriptResponse()
+    void shouldNotAlertOnSuspiciousCommentInJavaScriptResponse()
```

Comment:
`shouldNotAlertOnSuspiciousCommentInJavaScriptResponse` and `shouldAlertOnSuspiciousCommentInJavaScriptResponse` are conflicting, should it or not? I think the first should be better named.

Reply:
`shouldNotAlertOnSuspiciousValuesInJavaScriptResponse` ?